### DISCUSSION TITLE
How to customize the row title of MCMC graph?

### DISCUSSION THREAD

**Discussion by YmcGW** [phoebe-project/phoebe2#670] [Questions]
I want to ask how to customize the row title of MCMC graph like L1V/L2V, L1B/L2B of this graph.
![MCMC1](https://user-images.githubusercontent.com/118886353/206706959-68e0825b-d4dd-463e-a058-5ff441f76187.png)

**Comment by kecnry** [comment]
I think the [Latex Representations in Distribution Plots Tutorial](http://phoebe-project.org/docs/latest/tutorials/latex_repr.ipynb) should cover what you need.   If not, the returned figure is a matplotlib figure object, so you can always dig into that object and modify any of the axes/labels/styling.
